resources/views/home.blade.php

```diff
@@ -0,0 +1,290 @@
+            </div>
+            <!-- <div class= "team_item">
+              <img src="C:\Users\ASUS\Desktop\課題\img\t2.jpg" alt="">
+               -->
+               <div class = "team_item">
+                <img src="img\t3.jpg" alt="">
+                <div class = "team_intro">
+                  <p>Aika Hamashaki</p>
+                  <p>UIT大学の教師</p>
+                </div>
+                <div class= "team_detail">
+                  <p>Aika</p>
+                  <p>教師</p>
+                  <p>やはりBentoryが紹介した通りだよね。フォーを食べてみてすごく美味しかったよ。皆さん食べてみてね！</p>
+                  <!-- <div class= "team_social">
+                    <div class = "team_social-item">
+                      <i class= "fab fa-facebook"></i>
+                    </div>
+                    <div class = "team_social-item">
+                      <i class ="fab fa-instagram"></i>
+                    </div>
+                  </div> -->
+                </div>
+              </div>
+            
+
+          </div>
+        </section>
+        
+        
+        <section class="testimonials"></section>
+        <section class="contact"></section>
+        <section class="address"></section>
+        
+        
+        <section class="footer">
+          <div class="container">
+          <!--Bắt Đầu Nội Dung Giới Thiệu-->
+          <div class="noi-dung about">
+              <h2>Bentoryについて</h2>
+              <p>Bentoryにはたくさんベトナム料理について紹介する投稿が乗っています。ベトナム料理のウェブに日本人にベトナムの伝統的な食べ物や地域の特産物を紹介できる。そのため、日本人はベトナムの魅力を感じてベトナムが日本人にとって身近くなれる。</p>
+              <!-- <ul class="social-icon">
+                  <li><a href=""><i class="fa fa-facebook"></i></a></li>
+                  <li><a href=""><i class="fa fa-twitter"></i></a></li>
+                  <li><a href=""><i class="fa fa-instagram"></i></a></li>
+                  <li><a href=""><i class="fa fa-youtube"></i></a></li>
+              </ul> -->
+          </div>
+          <div class="noi-dung contact">
+            <h2>連絡先</h2>
+            <ul class="info2">
+              <div class= "icon_text">
+                  <span><i class="fa fa-map-marker"></i></span>
+                  <span class="info_right">1道
+                      1くん, ホーチミン市
+                      ベトナム</span>
+              <br>
+              </div>
+              <div class= "icon_text">
+               
+                  <span><i class="fa fa-phone"></i></span>
+                  
+                  <a href="#">+84 123 456 789</a>
+                    
+              <br>
+              </div>
+      
+              <div class= "icon_text">
+              
+                  <span><i class="fa fa-envelope"></i></span>
+                  <a href="#">diachiemail@gmail.com</a>
+          
+            
+              </div>
+                
+                
+            </ul>
+        </div>
+        </div>
+        </section>        
+    </div>
+    <script>
+        $(function(){
+        $('.selectpicker').selectpicker();
+    });
+    </script>
```

Comment:
```suggestion
    <script>
        $(function () {
            $('.selectpicker').selectpicker();
        });
    </script>
```
Bỏ script vào file js riêng nha